### DISCUSSION TITLE
Add jail reason parameter for Jail.Commands.OnJail so we can send Jail Reasons to other punishment plugins

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#7200] [Enhancement]
When OnJail happens, can we please be able to send the jail reason into this command, so we can /warn to keep track of all jails in Banmanager. Currently I cant find a way to send the reason from a jail when running the command.

**Comment by mrfloris** [comment]
You mean in the CMI-API ? I can't find onJail in there. 
Or do you mean you wish eventCommands.yml has an `onJail:` event you could add commands to?
Or do you mean if you run the /jail command? 
If it's the latter, turn off jail in alias.yml, then cmi aliaseditor and add a new command called jail. add sub commands to it that take the 'params' so when you type /jail bobby 5 x-ray
then $1= bobby, $2=5 $3=xray
You can use $3 like /otherplugin punished $1 $3 to give the other plugin command the name and reason.
